zhaquirks/tuya/tuya_smoke.py

```diff
@@ -60,8 +64,22 @@ def _update_attribute(self, attrid, value):
     .applies_to("_TZE284_rccxox8p", "TS0601")
     .applies_to("_TZE200_vzekyi4c", "TS0601")
     .applies_to("_TZE204_vawy74yh", "TS0601")
+    .tuya_smoke(dp_id=1)
+    .skip_configuration()
+    .add_to_registry()
+)
+
+(
+    TuyaQuirkBuilder("TZE200_0zaf1cr8", "TS0601")
     .applies_to("_TZE284_0zaf1cr8", "TS0601")
     .tuya_smoke(dp_id=1)
+    .tuya_binary_sensor(
+        dp_id=14,
+        attribute_name="battery_low",
+        translation_key="battery_low",
+        fallback_name="Battery low",
+    )
+    .tuya_battery(dp_id=15, battery_type=BatterySize.CR123A, battery_qty=1)
```

Comment:
But if we have a full battery percentage, why do we also have the "Battery low" sensor? I think we can just remove the latter one then. Or does only one work sometimes?

Reply:
z2m exposes both. I'm torn on the best method, if we don't expose it, we need to add a `tuya_dp` anyway, so we don't fill the log with errors when it reports an unknown datapoint. But as you said, it's a useless sensor.